st2common/tests/integration/test_register_content_script.py

```diff
@@ -144,5 +144,5 @@ def test_register_setup_virtualenvs(self):
         # All packs
         cmd = BASE_CMD_ARGS + ['--register-setup-virtualenvs', '--register-no-fail-on-failure']
         exit_code, stdout, stderr = run_command(cmd=cmd)
-        self.assertTrue('Setup virtualenv for 8 pack(s)' in stderr)
+        self.assertTrue('Setup virtualenv for 10 pack(s)' in stderr)
```

Comment:
Haha. I thought it's written like this on purpose, to manually check the pack number or something. 😛